lib/cli.js

```diff
@@ -124,9 +125,14 @@ exports.run = () => {
 	if (cli.flags.tap && !cli.flags.watch) {
 		reporter = tapReporter();
 	} else if (cli.flags.verbose || isCi) {
-		reporter = verboseReporter();
+		reporter = verboseReporter({
+			disableColors: cli.flags.color !== undefined
```

Comment:
@sindresorhus I tried to do what you suggested with default in in meow and that works for my code, 
`
 {
	color: true
}
`
but I don`t understand whats going on with what`s currently in default and how to make it work together with the change you recommended. Tried a few different things, but I keep getting a failing test in cli.js.

https://github.com/avajs/ava/blob/master/lib/cli.js#L75